mojave/structures/storage/crates.dm

```diff
@@ -236,3 +236,15 @@
 	material_drop = /obj/item/stack/sheet/ms13/scrap
 	material_drop_amount = 2
 	projectile_passchance = 85
+
+/obj/structure/closet/crate/ms13/woodcrate/compact/broken
+	name = "\improper broken wooden crate"
+	desc = "A wood storage crate, robust and study to all except a crowbar. This one seems already broken into."
```

Comment:
Code should look like.

/obj/structure/closet/crate/ms13/woodcrate/compact/broken
	name = "\improper broken wooden crate"
	desc = "A wood storage crate, robust and study to all except a crowbar. This one seems already broken into."
    icon_state = "plain_crate-1"

So mappers know its the broken one

Reply:
will the changing base icon state affect picking random one? I'm having doubts it will work

Reply:
nah it should work fine, it gets randomised on initialization, changing the base icon would just let mappers know which of the crates are pre-broken

Reply:
The init icon randomization is going off of the initial icon state, not the base icon state. If you changed the icon state right now it'd probably be invisible on init.